projects/isy-angular-widgets/src/lib/hauptfenster/hauptfenster.component.html

```diff
@@ -1,5 +1,5 @@
 <div class="isy-hauptfenster">
-  <div class="grid grid-nogutter">
+  <div class="grid grid-nogutter" role="banner" aria-label="banner">
```

Comment:
The <header></header> tag can be used here. Then role="banner" has be removed, as this is implied by header. Header already defines the role of the element as a banner. An additional aria-label="banner" would only provide redundant information here, as "banner" is not descriptive enough to explain the content or purpose of the banner. Better to delete aria-label="banner" at this point.